instrumentation-core/spring/spring-webmvc-3.1-core/spring-webmvc-3.1-core.gradle

```diff
@@ -0,0 +1,52 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_7
+}
+
+group = 'io.opentelemetry.instrumentation'
+version = '0.0.1-SNAPSHOT'
+
+apply from: "$rootDir/gradle/java.gradle"
+apply plugin: 'muzzle'
+
+muzzle {
+  pass {
+    group = 'org.springframework'
+    module = 'spring-webmvc'
+    versions = "[3.1.0.RELEASE,]"
+    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
+    skipVersions += '3.2.1.RELEASE' // missing a required class.  (bad release?)
+    extraDependency "javax.servlet:javax.servlet-api:3.0.1"
+    assertInverse = true
+  }
+
+  // FIXME: webmvc depends on web, so we need a separate integration for spring-web specifically.
+  fail {
+    group = 'org.springframework'
+    module = 'spring-web'
+    versions = "[,]"
+    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
+    extraDependency "javax.servlet:javax.servlet-api:3.0.1"
+  }
+}
+
+sourceCompatibility = '7'
+
+dependencies {
+  // copy-paste from instrumentation-core-aws-sdk:aws-sdk-2.2-core:
+  // TODO(anuraaga): We currently include common instrumentation logic like decorators in the
+  // bootstrap, but we need to move it out so manual instrumentation does not depend on code from
+  // the agent, like Agent.
+  api project(':auto-bootstrap')
+
+  // TODO(mabdinur):
+  // Move servlet-common to instrumentation-core
+  compileOnly project(':instrumentation:servlet:servlet-common')
```

Comment:
I need this getter: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/servlet/servlet-common/src/main/java/io/opentelemetry/auto/instrumentation/servlet/HttpServletRequestGetter.java

Reply:
I wanted to keep this pull request small. I plan to extract the servlet-common project to instrumentation-core in my next pull request.

Reply:
Take a look at #601. Probably I have done there exactly this.

Reply:
Awesome. I noticed that you moved (HttpServletRequestGetter)[https://github.com/iNikem/opentelemetry-java-instrumentation/blob/servlet-common/instrumentation-core/servlet/src/main/java/io/opentelemetry/instrumentation/servlet/HttpServletRequestGetter.java] to the core module. I'll update this dependency to map to :instrumentation-core:servlet:servlet-common when your change gets in. Keep an eye on this in case my change gets in first

Reply:
hey @mabdinur, I just merged #601 👍

Reply:
@trask Thanks. I just rebased from master and updated this dependency. This should be good to go